website/homepage/Dockerfile

```diff
@@ -14,15 +14,15 @@ RUN npm i pnpm@6 -g
 WORKDIR /gradio/ui
 ENV NODE_OPTIONS="--max-old-space-size=4096"
 RUN pnpm i
-RUN pnpm build:website
```

Comment:
This is why you can't get the demos to show up locally . `build:website` is a "production build" which makes you pull assets from the CDN. However, the assets corresponding to this build are never pushed to the CDN. We're also not setting a `GRADIO_VERSION` env variable.

From the vite-config:

```
export default defineConfig(({ mode }) => {
	const CDN_URL = mode === "production:cdn" ? CDN : "/";
	const production =
		mode === "production:cdn" ||
		mode === "production:local" ||
		mode === "production:website";
	const is_cdn = mode === "production:cdn" || mode === "production:website";
```

Using the local build means you pull the assets locally and that seems ok to me since we build the assets locally and use local urls to embed the demos.

Honestly confused how `gradio.app` is not facing this exact same problem lol.